index.html

```diff
@@ -1124,15 +1123,9 @@ <h3>Contexts</h3>
           </dd>
         </dl>
         <p class="note">
-Though this specification requires that a <code>@context</code> <a>property</a>
-be present, it is not required that the value of the <code>@context</code>
-<a>property</a> be processed using JSON-LD. This is to support processing using
-plain JSON libraries, such as those that might be used when the
-<a>verifiable credential</a> is encoded as a JWT. All libraries or processors
-MUST ensure that the order of the values in the <code>@context</code>
-<a>property</a> is what is expected for the specific application. Libraries or
-processors that support JSON-LD can process the <code>@context</code>
-<a>property</a> using full JSON-LD processing as expected.
+This specification requires for a <code>@context</code> <a>property</a>
+to be present, and it is required that the value of the <code>@context</code>
+<a>property</a> is processed using JSON-LD.
```

Comment:
Well, we should avoid possibly deviating from the JSON-LD spec itself:

> JSON-LD is used in many specifications as the specified data format. However, there is also a desire to allow some JSON-LD contents to be processed as plain JSON, without using any of the JSON-LD algorithms. Because JSON-LD is very flexible, some terms from the original format may be locally overridden through the use of [embedded contexts](https://www.w3.org/TR/json-ld11/#dfn-embedded-context), and take a different meaning for JSON-LD based implementations. On the other hand, "plain JSON" implementations may not be able to interpret these [embedded contexts](https://www.w3.org/TR/json-ld11/#dfn-embedded-context), and hence will still interpret those terms with their original meaning. To prevent this divergence of interpretation, JSON-LD 1.1 allows term definitions to be protected.

https://www.w3.org/TR/json-ld11/#protected-term-definitions

We can instead just reference JSON-LD directly to make it very clear that that spec is what is to be used to understand `@context`. Perhaps like this:

```suggestion
This specification requires for a <code>@context</code> <a>property</a>
to be present; this property is defined by [[JSON-LD]].
```

Reply:
I really like the suggestion! Thanks.